xmtp_mls/src/storage/encrypted_store/db_connection.rs

```diff
@@ -18,38 +25,77 @@ pub type DbConnection = DbConnectionPrivate<sqlite_web::connection::WasmSqliteCo
 // Do not derive clone here.
 // callers should be able to accomplish everything with one conn/reference.
 #[doc(hidden)]
+#[derive(Clone)]
```

Comment:
when do we clone the db connection?

Reply:
Ah, there was a moment where I was cloning it, but that implementation is gone. Let me clean it out.